assets/js/editor/editor.js

```diff
@@ -893,6 +895,19 @@ class Toolbar {
   }
 }
 
+function debounce(func, wait) {
```

Comment:
Could you please add a little doc commend describing what the function does? Basically, the description given in the PR would be a good fit.